lib/_stream_writable.js

```diff
@@ -707,6 +707,13 @@ Object.defineProperty(Writable.prototype, 'destroyed', {
   }
 });
 
+Object.defineProperty(Writable.prototype, 'writableObjectMode', {
+  enumerable: false,
+  get() {
+    return this._writableState && this._writableState.objectMode;
```

Comment:
Maybe return `false` instead of `undefined` if `this._writableState === undefined` when the stream wasn't instantiated ?

Refs: https://github.com/nodejs/node/pull/27190#discussion_r275705847